config/webpack/environment.js

```diff
@@ -7,10 +7,18 @@ dotenvFiles.forEach((dotenvFile) => {
   dotenv.config({ path: dotenvFile, silent: true })
 })
 
+// Strings needs to be interpolated and single quoted
```

Comment:
Interesting! does this cause issues if we have an apostrophe or anything in our env vars?